docs/api-reference/data-fetching/getInitialProps.md

```diff
@@ -8,7 +8,7 @@ description: Enable Server-Side Rendering in a page and do initial data populati
 >
 > If you're using Next.js 9.3 or newer, we recommend that you use `getStaticProps` or `getServerSideProps` instead of `getInitialProps`.
 >
-> These new data fetching methods allow you to have a granular choice between static generation and server-side rendering. Learn more on the documentation for [Pages](/docs/basic-features/pages.md) and [Data Fetching](/docs/basic-features/data-fetching.md).
+> These new data fetching methods allow you to have a granular choice between static generation and server-side rendering. Learn more on the documentation for [Pages](/docs/basic-features/pages.md) and [Data Fetching](/docs/basic-features/data-fetching/data-fetching.md).
```

Comment:
Should this just be /index?

Reply:
There was a reason I did this, but I don't remember now. But good shout